one/one/main.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+using namespace std;
```

Comment:
in our code, you'll likely see less using, and more actual explicit references to namespaces.

cout -> std::cout